_posts/2017-07-06-React-Native-for-iOS-devs.markdown

```diff
@@ -2,7 +2,7 @@
 layout: epic
 title: Intro to React Native for an iOS Developer
 date: 2017-07-06
-categories: [Technology, emission, reaction, react-native, react, javascript]
+categories: [Technology, emission, reaction, reactnative, react, javascript]
```

Comment:
I think that with the hyphen is technically more correct, but we’ve already been linking to the `reactnative` category in the past, so better to keep using that.

Reply:
Specifically for the react native one, we have the series https://artsy.github.io/series/react-native-at-artsy/ which is a better link, but yeah, consistency is 👍

Reply:
Oh lol, that’s definitely the link I was actually thinking of. Thanks!